### DISCUSSION TITLE
QTime: adding a prop to manage both emit model-value behaviors (Qv1 and Qv2)

### DISCUSSION THREAD

**Discussion by flopail** [quasarframework/quasar#13873] [Ideas / Proposals]
Hi, my feature request is in relationship with this comment : https://github.com/quasarframework/quasar/issues/13562#issuecomment-1169799907
In a few words, the model value is updated once we both modify hours and minutes in Quasar v1. In v2, model value is updated when we select hours, without awaiting minute selection.

I'm upgrading my app from Quasar v1 to v2. I use a QTime with a QInput, and a QPopupProxy with an auto-close function on event update;model-value. With v2 behavior, popup closes once hours are selected and before minute selection. I think it could be good to have a prop (a boolean for example) to manage both cases (v1 behavior and v2 behavior).

I searched for a similar feature request and I found this one, which is not so far from mine, but maybe larger because in relationship also with QDate: https://github.com/quasarframework/quasar/discussions/12166

Thank you for your feedback.

**Comment by 54mu3l** [comment]
Two people - same thought - **at the same time**. @flopail 

Seems like we were both writing simultaneously an Idea / Proposal (when I started writing, your disussion didn't exist yet):
- https://github.com/quasarframework/quasar/discussions/13874
- https://github.com/quasarframework/quasar/discussions/13875

I have the same use case. I need my QPopupProxy to auto-close.
